lib/report_portal/cucumber/parallel_report.rb

```diff
@@ -30,8 +33,8 @@ def initialize
         end
       end
 
-      def done(desired_time = ReportPortal.now)
-        end_feature(desired_time) if @feature_node
+      def test_run_finished(_event, desired_time = ReportPortal.now)
```

Comment:
This change is related to https://github.com/reportportal/agent-ruby/pull/45 but it was done here to prevent a conflict

Reply:
#45 was merged. Resolve the issue.